src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -50,6 +51,9 @@
 namespace osvr {
 namespace pluginhost {
     static const auto PLUGIN_HOST_LOGGER_NAME = "PluginHost";
+#ifdef _MSC_VER
+    static const auto PLUGIN_HOST_DEBUG_SUFFIX = ".debug";
```

Comment:
Why not set `PLUGIN_HOST_DEBUG_SUFFIX` to `""` on non-MSVC platforms to remove an `#ifdef` block later?